File: Go/10b.go
package main

import (
	"fmt"
	"math"
)

//lint:ignore U1000 //

/*
func main() {
	tenb()
}

//*/

func tenb() {
	var a, b, c, S, L, H float64
	fmt.Scan(&a, &b, &c)
	c = c * math.Pi / 180 //degree->radian
	S = a * b * math.Sin(c) / 2
	L = a + b + math.Sqrt(math.Pow(a, 2)+math.Pow(b, 2)-2*a*b*math.Cos(c))
	H = b * math.Sin(c)
	fmt.Println(S, L, H)
}

File: Python/10b.py
import math
a,b,c=(float(x) for x in input().split())
c = c * math.pi / 180 #degree->radian
print(a*b*math.sin(c)/2)
print(a+b+((a**2)+b**2-2*a*b*math.cos(c))**0.5)
print(b*math.sin(c))